### PR TITLE
Switch from Travis to Circle

### PR DESCRIPTION
**Pull request by nmiyake**
Fix issue #49

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,18 +0,0 @@
-sudo: false
-language: java
-jdk:
-- oraclejdk7
-env:
-  global:
-  - secure: IE/mfL1X/GtHnU5HtxL24u/SSkoTXnBBHbtfIpyjYwgbzGhPuq9qn9nEOBNb43ePaPGO+PDMz9sxDxHSIwBgbomEmq3dvQ9j+2jOx7MhmtCVr8Jz9x0xhug4cVPhaMVUZDa+z3qfSzbDbdtRKvuAzWHhwnbS9gQaGn1oIR2++5tXGZk+LVyvDO5/wbR6EcMxDQf0ZmUde/q0QtDmkIinS5r09ydUhsJE1wCUzyiOhx/kErnGmGD6buI/AgiZeGsGouonCykjrlNkoZkgmHobu/eCfbZPxUY9AkoJxORiLXZQ4GYH3KAvJlo5OZo99W8nw7wgnKZLJVkQD/sZzz3BCNlFQj41MroGUECm+fAq1hCDkXM1AmyFj9ZDQGyeb5+uEhf37mCN0TnUP6PNTTr+g/CqhQK67GDXM0i8LLKpEa/KtRuTtYm+gQ1VWJFYnxTx121XpEW8RKooqMbnBtOGngZqDqYb/8UjXg97logHThPMzpx3pkmDKlI+eB6wKi8VuBcMMIMU7K/pqUyWMqjK+JTAorPHJH17pCywsqsw0XghrIxjBjfTi7v8i9JxmV+9IadiaHhBj1WxJT3WnNEt7WEp4Q0ZxAL5ToiHT25ga55+9MHP9rJnqyunXon0q6MUoO3ubUmHU3wukCbakvZb7h8fEuCtMdzfA0TIZ7EsDIE=
```
Comment (nmiyake) [inline .travis.yml:7]: I imagine this needs to transfer over to Circle config, but not sure how to do so

File: .travis.yml [hunk 2]
```diff
@@ -1,18 +0,0 @@
...
-  - secure: KgQpjq/B3jLeEX1yaOWDQwYtIhPLtxsjbme83W740/bWBqjJ28NOmHeiwXzN2hw4pVMpz4LXn1jgB6Z38W91xsLsUsM5fINiaS27Q87VOE7B8fbJT5KImExFOcERdFqoCLtz4q2GA6DKbzbe8a5n9WkZT/EpuIPNgVtjlMtOP9K3A9+iuIPpbLgyE+j+LNQJINzxJE9BrN3qasTsD3wHQe7ZjEaUgYLHs/U+IbHXBG2IP4uXWMgk33BpNAK10E9H7W3xk4vIcFsvB5sLjpjKD/4mD1Ltmm/gkl3DwNEVmOxcDXCHHH5X0Rcf2gb41AbqiyUOs6Au/lkPF1lX+6bFsFF/gTIVuXxv3K3zUmRlRsdsd7HoAXp0sP/9GwrDs8ps/8MVPoaWKZu4IkNNPWBVboGRQmmGg5043mAyKhKYh/hygKgcHQpi8x4N7KNozLmbEagttbait66E87JAZNozGOKjEsK0G3Mvx64ISI9xmsIzPeqfLJDLkfnUAMODeR+JSYlTs1jYy3VPlx9X6uBGO6+qcySPZ7F/WAUgna1UwjVhbZM/hZoJU/1auYtpxiDOPm/ogHA/oleDALaO0cP1lqp6bV9PO/1/hpJWm/PG0C813UelykaF4VPqMmMR1C9Ob/GUzgpXB5rYjJKsPAVeiu093tkVNsjJ/rrmMmV2Tj0=
-after_success:
-- "./gradlew coveralls -PisCI"
```
Comment (nmiyake) [inline .travis.yml:10]: Similar here

File: circle.yml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    - http-clients/build/libs
+    - http-servers/build/libs
+    - ssl-config/build/libs
+
+test:
+  override:
+    - ./gradlew build
+
+deployment:
+  release:
```
Comment (nmiyake) [inline circle.yml:19]: Other projects have a "develop" block that runs a "publishToArtifactory" Gradle task, but this project doesn't have that task

**Comment by nmiyake** [comment]
Low-pri, but submitting to get the ball rolling in case it helps. Verified that the basics of the `circle.yml` should work on my own fork. However, it probably is missing some necessary things, and obviously is not valid until/unless Circle is turned on for the project as a whole.

**Comment by nmiyake** [comment]
@markelliot , thanks for enabling permissions to this on Circle. Added the project, and looks like builds/passes using Circle (although some publishing stuff will probably need to be touched up).

@uschi2000 for SA as well.

**Comment by markelliot** [comment]
Yeah, we need to copy over the env vars, @uschi2000, do you have those for this project? (I don't think I set this up)

**Comment by uschi2000** [comment]
Yep, I got them.

**Comment by uschi2000** [comment]
I can take this from here. Thanks Nick!

**Comment by uschi2000** [comment]
closing in lieu of https://github.com/palantir/http-remoting/pull/60

**Pull request closed without merging**